wakatime-mode.el

```diff
@@ -169,12 +169,26 @@ the wakatime subprocess occurs."
 (defun wakatime-client-command (savep)
   "Return client command executable and arguments.
    Set SAVEP to non-nil for write action."
-  (format "%s%s--file \"%s\" --plugin \"%s/%s\" --time %.2f%s%s"
+  (format "%s%s--file \"%s\" --plugin \"%s/%s\" --language \"%s\" --time %.2f%s%s"
     (if (s-blank wakatime-python-bin) "" (format "\"%s\" " wakatime-python-bin))
     (if (s-blank wakatime-cli-path) "wakatime " (format "\"%s\" " wakatime-cli-path))
     (buffer-file-name (current-buffer))
     wakatime-user-agent
     wakatime-version
+    ;; --language LANGUAGE Optional language name. If valid, takes priority over auto-detected language.
```

Comment:
Is the major mode always accurate? If not, we should use `--alternate-language`.